Source/Kesmai.Server/Game/Items/Rarity.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Drawing;
+
+namespace Kesmai.Server.Items
+public static class Rarity
+{
+	public static string GetRarity(int rarity.Clamp(0,5))
+	{
+		var rareText = "Common";
+		switch (rarity)
+		{
+			case 0:
+				break;
+
+			case 1:
+				rareText = "Uncommon";
```

Comment:
Two things here, should these be localized?  And should we come up with some more in theme names?  Somthing like worn, fine, exquisite plus a couple more on theme descriptions?